ethexe/contracts/test/POC.t.sol

```diff
@@ -98,9 +100,22 @@ contract POCTest is Base {
         depositInto(vaults[1], 10_000);
         depositInto(vaults[2], 10_000);
         rollBlocks((eraDuration - electionDuration) / blockDuration);
+
+        SigningKey _signingKey = FROSTOffchain.newSigningKey();
+        Vm.Wallet memory _publicKey = vm.createWallet(_signingKey.asScalar());
+
+        // TODO: makeElectionAt should also return Gear.AggregatedPublicKey
```

Comment:
put now some placeholder

Reply:
`SigningKey _signingKey = FROSTOffchain.newSigningKey();` and there is this placeholder. Instead of doing DKG we just use single `SigningKey`.